vladyslav.taran/Homework_first/task1/hash.rb

```diff
@@ -0,0 +1,2 @@
+p hash = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234', yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+p hash.select {|key|key.to_s.downcase.start_with? "yes"}.size
```

Comment:
Your code can find only 3 of 5 keys which start with 'yes'.
You miss 2 keys that start with whitespace `' yesterday1 '` and `' yesteryear '`
Try to find a method in ruby which can remove whitespace from a string.
If you don't find this method, let me know.